README.md

```diff
@@ -39,3 +39,8 @@ This package is available under the [MIT license](LICENSE).
 [coveralls_stable_link]: https://coveralls.io/github/sonata-project/SonataCoreBundle?branch=3.x
 [coveralls_unstable_badge]: https://coveralls.io/repos/github/sonata-project/SonataCoreBundle/badge.svg?branch=master
 [coveralls_unstable_link]: https://coveralls.io/github/sonata-project/SonataCoreBundle?branch=master
+
+[scrutinizer_stable_badge]: https://scrutinizer-ci.com/g/sonata-project/SonataCoreBundle/badges/quality-score.png?b=3.x
```

Comment:
this looks wrong

Reply:
Should it be master?

Reply:
yes and it is currently available, maybe your branch was outdated? 🤔

Reply:
ah sorry, no it isn't available in `master`, but yes it should be `master` in the URL

Reply:
But it says "stable", and master is unstable.

Reply:
argh you are right, sorry for confusing.

Everything is fine 👍